include/triton/Conversion/TritonGPUToLLVM/TargetInfoBase.h

```diff
@@ -10,6 +10,23 @@ class TargetInfoBase {
   virtual bool supportMaximumMinimum() const = 0;
   virtual Value callBallotOp(ConversionPatternRewriter &rewriter, Location loc,
```

Comment:
rename too ?

Reply:
Is `ballot` or `voteBallot` a better name ?

Reply:
I like `ballot`